Sources/CodeEditTextView/TextLine/LineFragmentView.swift

```diff
@@ -42,15 +42,6 @@ final class LineFragmentView: NSView {
         }
         context.saveGState()
 
-        context.setAllowsAntialiasing(true)
-        context.setShouldAntialias(true)
-        context.setAllowsFontSmoothing(false)
-        context.setShouldSmoothFonts(false)
-        context.setAllowsFontSubpixelPositioning(true)
-        context.setShouldSubpixelPositionFonts(true)
-        context.setAllowsFontSubpixelQuantization(true)
-        context.setShouldSubpixelQuantizeFonts(true)
-
```

Comment:
Was this removal intentional? I thought we were just re-enabling font smoothing (which makes perfect sense to me, I might've just been throwing things at the wall).

Reply:
It was, although I did question what the sub pixel lines were for. I can add them back in if they are necessary. But if you were just throwing stuff at the wall and they aren't needed, lets keep out unnecessary code.

Reply:
Sub pixel rendering effectively increases the resolution of the displaying device, it should increase the clarity of rendered text. However it does decrease in effectiveness when there's less contrast with its background (not black and white) so there's a trade off there.

Antialiasing generally smooths things, and subpixel quantization could be turned off as that actually decreases positioning accuracy slightly with the benefit of performance and higher quality bitmaps for drawn text.

I'd say let's keep it, I can add a comment to that method on the tradeoffs/benefits of each and we could revisit it if needed.